src/createServer.js

```diff
@@ -11,48 +11,26 @@ function createServer() {
     const pathName = normalizeUrl.pathname;
     const arrayFromPathName = pathName.slice(1).split('/');
 
-    if (arrayFromPathName.includes('..')) {
-      res.writeHead(400, {
-        'Content-Type': 'text/plain',
-      });
-
-      res.end(
-        JSON.stringify({
-          error: 'Path to file should be like this /file/folderName/filename',
-        }),
-      );
-
-      return;
-    }
```

Comment:
you need to just change condition not delete entirely
https://github.com/mate-academy/node_static-files-server/pull/233#discussion_r1721472706